File: src/react-frontend/src/app/types/models/game/gameSearchItem.ts
import { GameStatus } from 'app/types/enums/gameStatus';
import { UserDto } from '../user/userDto';

export type GameSearchItem = {
  id: string;
  code: string;
  name: string;
  isPrivate: boolean;
  status: GameStatus;
  users: UserDto[];
};

File: src/Domain/Models/Games/GameSearchItem.cs
﻿using Domain.Enums;
using Domain.Models.Users;

namespace Domain.Models.Games;

public class GameSearchItem
{
    public string Id { get; set; }
    public string Code { get; set; }
    public string Name { get; set; }
    public bool IsPrivate { get; set; }
    public GameStatus Status { get; set; }
    public List<UserDto> Users { get; set; }
}
